terraform/environments/corporate-staff-rostering/locals_development.tf

```diff
@@ -30,6 +30,38 @@ locals {
           server-type = "base-ol-8-5"
         }
       }
+    
+      web-dev-server-ami	 = {
+        config = merge(module.baseline_presets.ec2_instance.config.default, {
+          ami_name          = "web-dev-server-ami"
+          ami_owner         = "self"
+          availability_zone = "${local.region}a"
+          ebs_volumes_copy_all_from_ami = false
+          instance_profile_policies     = concat(module.baseline_presets.ec2_instance.config.default.instance_profile_policies, ["CSRWebServerPolicy"])
```

Comment:
can you please remove this instance_profile_policies line? 

at the moment you won't be able to use it as the 'user' end i.e. you and me don't have permissions to use it yet. 

it's a dev asg so we can always add this later, also I discovered you can't delete policies without first deleting everything it might be connected to...

Reply:
removed